db/app.controller.js

```diff
@@ -23,13 +23,27 @@ exports.getArticles = (req, res, next) => {
 }
 
 exports.getArticleById = (req, res, next) => {
+  if (isNaN(req.params.article_id)) {
+    return next({ status:400, message: "Invalid Article ID !"})
+  }
```

Comment:
This might not work as you expect - `req.params.article_id` will always come back as a string, even if the value is numeric. However, this check may not be necessary - currently, do you have a test that motivates this if statement being included at all?